### PR TITLE
Add WiseBanyan

### PR DESCRIPTION
**Pull request by januszhou**

### CODE REVIEW

File: _data/investing.yml [hunk 1]
```diff
@@ -143,3 +143,12 @@ websites:
...
       img: wealthfront.png
       tfa: no
+      
+    - name: WiseBanyan
+      url: https://www.wisebanyan.com/
+      twitter: WiseBanyan
+      img: wisebanyan.png
+      tfa: Yes
+      sms: Yes
+      software: Yes
```
Comment (Carlgo11) [inline _data/investing.yml]: I see the doc mention SMS authentication but not software.

**Comment by januszhou** [comment]
You right, changed

File: _data/investing.yml [hunk 2]
```diff
@@ -143,3 +143,11 @@ websites:
       twitter: wealthfront
       img: wealthfront.png
       tfa: no
+
+    - name: WiseBanyan
+      url: https://www.wisebanyan.com/
+      twitter: WiseBanyan
```
Comment (Carlgo11) [inline _data/investing.yml]: Twitter isn't needed as they already support TFA.

**Comment by januszhou** [comment]
Cool, already removed

**Comment by Carlgo11** [comment]
Nice. Great work :smiley:

**Comment by mxxcon** [comment]
Reviewed 1 of 2 files at r1, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 2 unresolved discussions, all commit checks successful.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/jdavis/twofactorauth/1202)

**Comment by mxxcon** [comment]
Reviewed 1 of 2 files at r1, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 2 unresolved discussions, all commit checks successful.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/jdavis/twofactorauth/1202)

**Comment by swrobel** [comment]
Was just about to add this. Thanks for beating me to it!

**Comment by januszhou** [comment]
Just wondering, what's the deploy timeline?

**Comment by Carlgo11** [comment]
@januszhou https://github.com/jdavis/twofactorauth/wiki/FAQ_Merge

**Pull request merged**